voteswap/templates/base.html

```diff
@@ -32,8 +32,10 @@
         </div>
       </div>
     </nav>
-    {% block main %}
-    {% endblock %}
+    <div class="container">
```

Comment:
Hm, you're right. I was misremembering. Change it to `container-full` though, and remove it from the `voteswap/templates/landing_page.html` template, please.